packages/components/NEW-COMPONENT-CHECKLIST.md

```diff
@@ -5,13 +5,74 @@ This is the quest issue/engineering checklist for the COMPONENT_NAME Component.
-- [Design System Website](https://design-system-website.vercel.app/?path=/story/example-introduction--page) (storybook of storybooks): use for reference, to consider existing features that we might need to replicate in the component.
+- [Component Audit](https://www.figma.com/file/h3LtloARmbYVwh7z6XWHVb/Component-Audits-FY23?node-id=0%3A1): This audit should be completed by ambassadors before any work gets started.
+- [Component Requirement Document (CRD)](url_here): This file should exist before any design explorations or code is written.
+- ["System of Systems" Storybook](https://design-system-website.vercel.app/?path=/story/example-introduction--page) (storybook of storybooks): use for reference, to consider existing features that we might need to replicate in the component.
+
+
+## Design Checklist 
+The design checklist has 5 parts: 
+1. creating the branch
+2. design explorations and iterations
+3. documentation
+4. release
+5. testing
+
+The design process is largely cyclical, so some of these steps may need to be repeated until approval.
+
+### Component Creation
+- [ ] [create a branch](https://help.figma.com/hc/en-us/articles/360063144053-Create-branches-and-merge-changes#Create_a_branch) in the [product components ui kit](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components)
+- [ ] add a new page for the component, as needed
+- [ ] copy & paste the [stickersheet template](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components?node-id=2279%3A26800) found in UI Helpers to the new page
+  - [ ] update the status badge to "In progress"
+  - [ ] change "{DOC TYPE}" to "STICKERSHEET"
+  - [ ] change "{Component name}" to the name of the component
+  - [ ] delete the sunset notice
+- [ ] **design**
+  - [ ] work through design explorations, providing 2-3 visual directions based on existing [foundations](https://www.figma.com/file/oQsMzMMnynfPWpMEt91OpH/HDS-Product---Foundations?node-id=2916%3A4) and patterns, as well as the findings from the audit and requirements from the CRD
+  - [ ] review with appropriate stakeholders (typically HDS designers, engineering partner, ambassadors) and iterate
+  - [ ] review for a11y considerations and iterate, as necessary
+  - [ ] review with design leadership and iterate, as necessary
+  - [ ] once approved, build out the design as a component
+    - ensure the variant and property names align with what was discussed/decided on with your engineering partner
+  - [ ] test the component with HDS designers and iterate, as necessary
+  - [ ] sync with your engineering partner (pseudo handoff)
+    - let them know it's been design approved and discuss any remaining questions (this should be an ongoing process as well)
+- [ ] **documentation**
+  - all documentation should be contained within a frame so it can be easily linked to
```

Comment:
```suggestion
  - all documentation should be contained within frames so it can be easily linked to
```
Minor suggestion to clarify that not everything needs to be within the same frame